fees/wise-lending-v2/index.ts

```diff
@@ -0,0 +1,44 @@
+import { FetchOptions, SimpleAdapter } from "../../adapters/types";
+import { ARBITRUM } from "../../helpers/chains";
+const feeMaangerContract = "0x90a022796798f9dbA1Da0f8645234B284d4E8EC6";
+
+const arbitrumPools = [
+    //aave
+    "0xe50fA9b3c56FfB159cB0FCA61F5c9D750e8128c8",
+    "0x6ab707Aca953eDAeFBc4fD23bA73294241490620",
+    "0x724dc807b04555b71ed48a6896b6F41593b8C637",
+    "0x82E64f49Ed5EC1bC6e43DAD4FC8Af9bb3A2312EE",
+    //native
+    "0xFd086bC7CD5C481DCC9C85ebE478A1C0b69FCbb9",
+    "0xaf88d065e77c8cC2239327C5EDb3A432268e5831",
+    "0x82aF49447D8a07e3bd95BD0d56f35241523fBab1",
+    "0xDA10009cBd5D07dd0CeCc66161FC93D7c9000da1",
+    "0x5979D7b546E38E414F7E9822514be443A4800529",
+];
+
+const fetch: any = async ({ api }: FetchOptions) => {
+    const feeAmounts = await api.multiCall({
+        abi: "function feeTokens(address) view returns (uint256)",
+        target: feeMaangerContract,
+        calls: arbitrumPools,
+    });
+
+    api.add(arbitrumPools, feeAmounts);
```

Comment:
seem it total value not daily

Reply:
if you query this endpoint daily then you get the values on daily basis

Reply:
so this adapter shows our current fees generated by the protocol that are not withdrawn, as soon as protocol withdraws the fees this global value of fees accumulated is reduced accordingly. @dtmkeng feel free to open our contract and see how the fees are handled. 

- query feeTokens() to see accumulated fees for the pool that are not withdrawn, query feeTokens() at any time to get the amount of fees protocol holds at each point in time. 

- elaborate about daily? contract provides fees for each pool at every given moment every time you make a snapshot of this data (daily, monthly or each hour is up to you) we give you data on chain about fees we've collected at each point available from the protocol.

Reply:
> seem it total value not daily

can you elaborate how it should be? did you check contract code? the fees reported are correct per pool, if they are withdrawn on daily basis call to this function would reflect that in value updated every time queried.

Reply:
daily value from contract function should like this https://github.com/DefiLlama/dimension-adapters/blob/master/fees/crv-usd.ts